dashboard/app/assets/javascripts/utils.js.erb

```diff
@@ -10,9 +10,14 @@ var addClickTouchEvent = function(element, handler) {
 };
 
 function onURLBlocked(base_url, callback) {
-  $("<img width=0 height=0>")
+  $('<img width="0" height="0">')
```

Comment:
I think there's a quirk of older IE versions where the inline size attributes are overridden by some general/implicit CSS rule.  [Here's a report of a similar issue on StackOverflow](http://stackoverflow.com/questions/12521716/internet-explorer-10-ignores-width-height-on-images).  Maybe I should just remove the inline attributes?